src/sass/base/_va.scss

```diff
@@ -224,6 +224,24 @@ hr {
   // TODO: clean up #content .main.interior a then remove !important
   text-decoration: none !important;
 }
+// Option for adding exit-icon to ghost buttons
+.usa-button-outline-exit.usa-button-primary[href^=http]::after {
```

Comment:
Was there a reason for using `::after` instead of setting the background image directly, like how it's done above?